crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2951,15 +2951,6 @@ impl AuthorityPerEpochStore {
                 }
             }
             checkpoint_roots.extend(roots.into_iter());
-            let pending_checkpoint = PendingCheckpointV2::V2(PendingCheckpointV2Contents {
-                roots: checkpoint_roots,
-                details: PendingCheckpointInfo {
-                    timestamp_ms: consensus_commit_info.timestamp,
-                    last_of_epoch: final_round && randomness_round.is_none(),
-                    checkpoint_height,
-                },
-            });
-            self.write_pending_checkpoint(&mut output, &pending_checkpoint)?;
 
             // Generate pending checkpoint for user tx with randomness.
```

Comment:
nit - I think the placement of this comment is now confusing because what gets written below is no longer only the randomness PendingCheckpoint, I had to read everything a few times to understand the reordering

Also the comment above on line 2933 talks about something that is no longer being done in that block?

Reply:
fixed!